test/inductor/test_torchinductor.py

```diff
@@ -12958,6 +12958,9 @@ def fn(x):
         self.common(fn, (x,), check_lowp=False)
 
     @config.patch(implicit_fallbacks=True)
+    @skip_if_cpp_wrapper(
```

Comment:
cc @desertfire @benjaminglass1 ..

Reply:
@shunting314 Can you open an issue for this and assign it to me? I'd like to take a look at this in the next week or two.

Reply:
sure: https://github.com/pytorch/pytorch/issues/151027